Hardit_Singh/bankAccount.rb

```diff
@@ -0,0 +1,59 @@
+class BankAccount
+	attr_reader	:accountNumber,
+				:userName,
+				:checkingBalance,
+				:savingBalance
+
+	def initialize(userName,balance=0)		
+		@accountNumber = genAccountNumber
+		@userName = userName
+		@checkingBalance = balance
+		@savingBalance = balance
+		@@noOfAccounts += 1

```

Comment:
Do you mean @@noOfAccounts? 
it doesnt give any errors. And I initiated it as @@noOfAccounts = 0.